eu.fortysixandtwo.chessclock.json

```diff
@@ -0,0 +1,67 @@
+{
+    "app-id" : "eu.fortysixandtwo.chessclock",
```

Comment:
Can you put a link to the gitlab of chess-clock on https://fortysixandtwo.eu/ ?

Reply:
I don't have a gitlab there.
The sources are living @ https://gitlab.gnome.org/World/chess-clock/

Reply:
The page says:

> Welcome to nginx!
>
> If you see this page, the nginx web server is successfully installed and working. Further configuration is required.

How do we know it's related? That's what you are being asked.

Reply:
I thought verification of domain ownership is done through placing something under `.well-known/foo/bar`.

> The page says:
> ...

yes and it has served me well > 7 years :innocent:

Reply:
I'm prepared to server whatever needs serving under https://fortysixandtwo.eu/.well-known/org.flathub.VerifiedApps.txt

https://docs.flathub.org/docs/for-app-authors/verification#website seems to suggest this is only done after the app has been submitted.

Reply:
we have seen many people submit app-id ignoring that they domain belonged to someone else. And really insisting on it. So we do check now as it can't be changed.